cw_monero/lib/api/account_list.dart

```diff
@@ -37,7 +37,8 @@ List<monero.SubaddressAccountRow> getAllAccount() {
   int size = monero.SubaddressAccount_getAll_size(subaddressAccount!);
   if (size == 0) {
     monero.Wallet_addSubaddressAccount(wptr!);
-    return getAllAccount();
+    monero.Wallet_status(wptr!);
+    return [];
```

Comment:
I don't think this is right? or can you explain why we will return an empty string after already adding an account?